pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -235,9 +235,31 @@ func ValidateFlowSchemaResourcePolicyRule(rule *flowcontrol.ResourcePolicyRule,
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("resources"), rule.Resources, "if '*' is present, must not specify other resources"))
 	}
 
+	if len(rule.Namespaces) == 0 {
+		allErrs = append(allErrs, field.Required(fldPath.Child("namespaces"), "resource rules must supply at least one namespace"))
+	} else if memberInList(flowcontrol.NamespaceEvery, rule.Namespaces...) {
```

Comment:
Since namespaces have two wildcards, the function name "hasWildcard" is potentially confusing here; I chose to invoke "memberInList" instead for the sake of clarity.